app/javascript/components/new-action-item/new-action-item.jsx

```diff
@@ -0,0 +1,90 @@
+import React, {useState, useContext, useMemo} from 'react';
+import {useMutation} from '@apollo/react-hooks';
+import {addActionItemMutation} from './operations.gql';
+import BoardSlugContext from '../../utils/board-slug-context';
+import UserContext from '../../utils/user-context';
+import style from './style.module.less';
+import {CardUser} from '../card-user';
+import {ActionItemEdit} from '../action-item-edit';
+
+const NewActionItem = ({users}) => {
+  const currentUser = useContext(UserContext);
+  const [isOpened, setOpened] = useState(false);
+  const [newActionItemBody, setNewActionItemBody] = useState('');
+  const [newActionItemAssigneeId, setNewActionItemAssigneeId] = useState('');
+  const [addActionItem] = useMutation(addActionItemMutation);
+  const boardSlug = useContext(BoardSlugContext);
+
+  const cancelHandler = (evt) => {
+    evt.preventDefault();
+    setOpened(!isOpened);
+    setNewActionItemBody('');
+  };
+
+  const submitHandler = async (evt) => {
+    evt.preventDefault();
+
+    const {data} = await addActionItem({
+      variables: {
+        boardSlug,
+        assigneeId: newActionItemAssigneeId,
+        body: newActionItemBody
+      }
+    });
+    if (data.addActionItem.actionItem) {
+      setNewActionItemBody('');
+    } else {
+      console.log(data.addActionItem.errors.fullMessages.join(' '));
+    }
+  };
+
+  const handleKeyPress = (evt) => {
+    if (navigator.platform.includes('Mac')) {
+      if (evt.key === 'Enter' && evt.metaKey) {
+        submitHandler(evt);
+      }
+    } else if (evt.key === 'Enter' && evt.ctrlKey) {
+      submitHandler(evt);
+    }
+
+    if (evt.key === 'Escape') {
+      setOpened(!isOpened);
+      setNewActionItemBody('');
+    }
+  };
```

Comment:
You could use helper here also https://github.com/cybergizer-hq/retrospective/pull/392/files#r684013767